src/App.js

```diff
@@ -2,12 +2,26 @@ import React from "react";
 
 import Bookings from "./Bookings";
 import "./App.css";
+import Heading from "./Heading";
+import TouristInfoCards from "./TouristInfoCards";
+import Footer from "./Footer";
+import Restaurant from "./Restaurant";
 
 const App = () => {
   return (
     <div className="App">
-      <header className="App-header">CYF Hotel</header>
+      <Heading />
+      <TouristInfoCards />
       <Bookings />
+      <Restaurant />
+
+      <Footer
+        address={[
```

Comment:
Do you think `address` is the best name for this prop? Only 1 string in the array is about the address, the others aren't. What do you think a better name could be?